nautobot/circuits/api/serializers.py

```diff
@@ -98,17 +97,15 @@ class Meta:
 
 class CircuitCircuitTerminationSerializer(WritableNestedSerializer, NotesSerializerMixin):
     url = serializers.HyperlinkedIdentityField(view_name="circuits-api:circuittermination-detail")
-    site = NestedSiteSerializer()
     location = NestedLocationSerializer(required=False, allow_null=True)
     provider_network = NestedProviderNetworkSerializer()
-    connected_endpoint = NestedInterfaceSerializer()
+    connected_endpoint = NestedCircuitTerminationSerializer()
```

Comment:
Hmm. Actually neither of these is "correct" - `connected_endpoint` is an alias to the generic foreign key `circuit_termination._path.destination`, which could be any subclass of `PathEndpoint`. It looks to me like this serializer should probably inherit from `nautobot.dcim.api.serializers.PathEndpointModelSerializerMixin` rather than implementing its own `connected_endpoint` logic.